diff --git a/package-lock.json b/package-lock.json
index f1c4f69..7ae7178 100644
--- a/package-lock.json
+++ b/package-lock.json
@@ -32,7 +32,6 @@
 				"react-dom": "^19.2.8",
 				"react-hook-form": "^7.76.1",
 				"react-hot-toast": "^2.6.0",
-				"react-icons": "^5.7.0",
 				"react-number-format": "^5.4.5",
 				"react-router": "^8.3.0",
 				"tailwind-merge": "^3.6.0"
@@ -4146,15 +4145,6 @@
 				"react-dom": ">=16"
 			}
 		},
-		"node_modules/react-icons": {
-			"version": "5.7.0",
-			"resolved": "https://registry.npmjs.org/react-icons/-/react-icons-5.7.0.tgz",
-			"integrity": "sha512-LBLy340Rzqy6+/yVhZKT3B/QpP1BZaesGqasf09HPOBzRarcDIFH0WwXlXQfE7q7ipxK4MSiC5DIBWURCny6fw==",
-			"license": "MIT",
-			"peerDependencies": {
-				"react": "*"
-			}
-		},
 		"node_modules/react-number-format": {
 			"version": "5.4.5",
 			"resolved": "https://registry.npmjs.org/react-number-format/-/react-number-format-5.4.5.tgz",
diff --git a/package.json b/package.json
index 516c314..b020e3b 100644
--- a/package.json
+++ b/package.json
@@ -34,7 +34,6 @@
 		"react-dom": "^19.2.8",
 		"react-hook-form": "^7.76.1",
 		"react-hot-toast": "^2.6.0",
-		"react-icons": "^5.7.0",
 		"react-number-format": "^5.4.5",
 		"react-router": "^8.3.0",
 		"tailwind-merge": "^3.6.0"
diff --git a/src/features/mileage/AddMilesForm.jsx b/src/features/mileage/AddMilesForm.jsx
index 589466d..7fbf123 100644
--- a/src/features/mileage/AddMilesForm.jsx
+++ b/src/features/mileage/AddMilesForm.jsx
@@ -35,7 +35,6 @@ export default function AddMilesForm() {
     formState: { isSubmitting },
   } = useForm({
     defaultValues: {
-      category: null,
       locations: [
         {
           country: null,
@@ -63,15 +62,13 @@ export default function AddMilesForm() {
         locations: [...data.locations].flat(),
       };
 
-      console.log({ data, payload });
-      return;
       await insertMileageEntry(payload);
       queryClient.invalidateQueries({ queryKey: ['miles'] });
 
       toast.success('Mileage entry successfully saved');
       reset();
     } catch (err) {
-      toast.error('Could not save your entry, try again`');
+      toast.error('Could not save your entry, try again');
       console.log(err);
     }
   };
@@ -89,9 +86,6 @@ export default function AddMilesForm() {
     if (errors.locations) {
       toast.error(errors.locations.message);
     }
-    if (errors.locations) {
-      toast.error(errors.locations.message);
-    }
     return;
   };
 
@@ -122,12 +116,7 @@ export default function AddMilesForm() {
           <div className='mb-4 flex justify-evenly pb-6'>
             {/* Date */}
             <div className='flex flex-col gap-0.5'>
-              <FieldLabel
-                className='text-md text-center'
-                htmlFor='initial-odometer'
-              >
-                Date
-              </FieldLabel>
+              <FieldLabel className='text-md text-center'>Date</FieldLabel>
               {/* DATE INPUT */}
               <DatePicker
                 label='Date'
@@ -325,7 +314,7 @@ export default function AddMilesForm() {
                             radioGroup: cn('flex justify-evenly'),
                             radioField: cn(''),
                             radioButton: cn(
-                              'flex cursor-pointer place-items-center gap-2 rounded-sm border border-gray-300 px-1.5 py-0.5 text-sm tracking-tighter text-gray-500 transition-all duration-90 hover:scale-105 active:scale-90 active:text-gray-400 data-focus-visible:ring-2 data-focus-visible:ring-emerald-500 data-focused-visible:ring-offset-3 data-hover:scale-105 data-hovered:text-gray-600 data-selected:border-emerald-600 data-selected:font-medium data-selected:text-emerald-700',
+                              'group flex cursor-pointer place-items-center gap-2 rounded-sm border border-gray-300 px-1.5 py-0.5 text-sm tracking-tighter text-gray-500 transition-all duration-90 hover:scale-105 active:scale-90 active:text-gray-400 data-focus-visible:ring-2 data-focus-visible:ring-emerald-500 data-focus-visible:ring-offset-3 data-hovered:scale-105 data-hovered:text-gray-600 data-selected:border-emerald-600 data-selected:font-medium data-selected:text-emerald-700',
                             ),
 
                             textInputContainer: cn(
@@ -355,6 +344,7 @@ export default function AddMilesForm() {
                   country: null,
                   region: null,
                   locality: null,
+                  category: null,
                 });
               }}
             >
diff --git a/src/features/mileage/MileageNavbar.jsx b/src/features/mileage/MileageNavbar.jsx
index 29e8e95..e2bbe0c 100644
--- a/src/features/mileage/MileageNavbar.jsx
+++ b/src/features/mileage/MileageNavbar.jsx
@@ -10,7 +10,7 @@ export default function Navbar() {
         }
         to='/'
       >
-        <img src={logo} className='h-8 w-auto' />
+        <img src={logo} alt='MileMetrics' className='h-8 w-auto' />
       </NavLink>
 
       <ul className='mr-1 flex gap-5 text-xs font-semibold text-slate-600'>
diff --git a/src/ui/Button.jsx b/src/ui/Button.jsx
index 0715c5f..30347cc 100644
--- a/src/ui/Button.jsx
+++ b/src/ui/Button.jsx
@@ -12,7 +12,10 @@ export default function Button({
     <BaseUIButton
       type={type}
       disabled={disabled}
-      className={twMerge(className)}
+      className={twMerge(
+        'rounded-xl border-2 border-slate-500 bg-slate-50 p-3 text-center text-sm font-semibold tracking-wider text-slate-500 transition-all duration-150 hover:cursor-pointer hover:bg-slate-300 hover:text-slate-600 focus:outline-none focus-visible:border-emerald-500 focus-visible:shadow focus-visible:ring-3 focus-visible:ring-emerald-500 active:scale-95 active:border-slate-400/75 active:bg-slate-100 active:text-slate-600/75',
+        className,
+      )}
       onClick={onClick}
     >
       {children}
diff --git a/src/ui/Calendar.jsx b/src/ui/Calendar.jsx
index 1842de7..d84b852 100644
--- a/src/ui/Calendar.jsx
+++ b/src/ui/Calendar.jsx
@@ -44,7 +44,7 @@ const Calendar = ({
         </Button>
         <CalendarHeading
           format={calendarHeaderFormat}
-          className={classNames.calendarHeading}
+          className={classNames.calendarHeader}
         />
         <Button slot='next' className={classNames.arrowRight}>
           <ArrowRight size={svgSize} />
diff --git a/src/ui/Fieldset.jsx b/src/ui/Fieldset.jsx
index d996801..4ad0aaa 100644
--- a/src/ui/Fieldset.jsx
+++ b/src/ui/Fieldset.jsx
@@ -23,7 +23,7 @@ const Fieldset = ({
         >
           {label}
           {description}
-          <div className='flex place-content-center'>{input}</div>
+          <div className="flex place-content-center">{input}</div>
         </Field>
       ))}
     </div>
diff --git a/src/ui/PopoverV2.jsx b/src/ui/PopoverV2.jsx
deleted file mode 100644
index 29b2843..0000000
--- a/src/ui/PopoverV2.jsx
+++ /dev/null
@@ -1,26 +0,0 @@
-import { useState } from 'react';
-import handleError from '../utils/handleError';
-import {
-  Popover as AriaPopOver,
-  DialogTrigger,
-} from 'react-aria-components/Popover';
-
-const Popover = ({ content, trigger }) => {
-  const [isOpen, setIsOpen] = useState(false);
-
-  if (!trigger)
-    handleError(`Popover: A 'trigger' element is required`, {
-      trigger,
-    });
-
-  return (
-    <DialogTrigger isOpen={isOpen} onOpenChange={() => setIsOpen(!isOpen)}>
-      {trigger}
-      <AriaPopOver onOpenChange={setIsOpen} isOpen={isOpen}>
-        {content}
-      </AriaPopOver>
-    </DialogTrigger>
-  );
-};
-
-export default Popover;
diff --git a/src/ui/RadioTile.jsx b/src/ui/RadioTile.jsx
index 3ba6030..1e8f8eb 100644
--- a/src/ui/RadioTile.jsx
+++ b/src/ui/RadioTile.jsx
@@ -84,10 +84,7 @@ const RadioTile = ({
             classNames.textInputContainer,
           )}
         >
-          <RadioButton
-            onFocus={() => customInputRef.current?.focus()}
-            className='select-none'
-          >
+          <RadioButton>
             <Pencil size={17} className={classNames.textInputSVG} />
           </RadioButton>
 
